OMCompiler/Compiler/Script/CevalScriptBackend.mo

```diff
@@ -3387,6 +3391,161 @@ algorithm
   end matchcontinue;
 end runFrontEndWork;
 
+
+/* -------------------------------------------------------------------
```

Comment:
This comment is a bit unnecessary, considering the name of the method

Reply:
Correct but i like visual dividers. It refers to all following functions, in this case probably still useless since it covers only two functions